README.md

```diff
@@ -3,7 +3,7 @@
 # What is GAP?
 
 GAP is a system for computational discrete algebra, with particular emphasis on
-Computational Group Theory. GAP provides a programming language, a library of
+computational group theory. GAP provides a programming language, a library of
```

Comment:
Agree - perhaps same should be done for the website and maybe manual, for consistency? (not necessarily in this PR)

Reply:
I don't see this appear in the manual, and obviously I can't fix it on the website with this PR.